src/Surfnet/StepupMiddleware/ApiBundle/Authorization/Service/InstitutionAuthorizationContextFactory.php

```diff
@@ -56,10 +58,9 @@ public function buildFrom(Request $request, InstitutionRoleSetInterface $roleReq
                 throw new InvalidArgumentException('Identity with id "%s" could not be found.');
             }
 
-            $institution = $request->get('actorInstitution');
-            $actorInstitution = new Institution($institution);
+            $institution = new Institution($actorInstitution);
 
-            // The identity that was returned from the service should match the institutio that was provided in the
+            // The identity that was returned from the service should match the institution that was provided in the
             // actorInstitution request parameter.
             if (!$actorInstitution->equals($actorIdentity->institution)) {
```

Comment:
$actorInstitution is now a string

~~$actorInstitution~~ $institution

Reply:
Thanks!